site/app/templates/forum/ForumBar.twig

```diff
@@ -31,8 +31,8 @@
 	{% endif %}
 
 	{% if thread_exists %}
-		<form id="search_block" style="float:right;position:relative;top:3px;right:5px;display:none;" method="post" action="{{ core.buildUrl({'component':'forum', 'page':'search_threads'}) }}">
-			<input type="text" size="35" placeholder="search" name="search_content" id="search_content"/>
+		<form id="search_block" style="float:right;position:relative;top:3px;right:5px;display:none;" method="post" action="{{ core.buildUrl({'component':'forum', 'page':'search_threads'}) }}" onsubmit="if($('#search_content').text().replace(/ /g,'').length == 0){ return false; }">
```

Comment:
Would it make more sense to just do:
```
return $('#search_content').text().trim().length !== 0;
```
?

Reply:
Probably we can use onchange  which will change normal value with trimmed value as at last trim function is somewhat applied in searching in some ways so let us apply before submission and It will also take care of whitespaces as trimming whitespaces will result into blank output

Reply:
@MasterOdin Is this Issue needed to be fix?